docs/executor_plugins.md

```diff
@@ -30,6 +30,15 @@ controller:
       value: "true"
 ```
 
+In case you want to modify a running `workflow-controller`:
```

Comment:
```suggestion
If you want to modify a running Controller:
```

- simplify, "in case" -> "if", per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- "controller" is how it is referred to in this doc and elsewhere, be consistent